examples/gluon_alexnet/README.md

```diff
@@ -0,0 +1,370 @@
+mv alexnet.params /tmp/models
+```
+4. Run the model-export tool on this folder.
+```bash
+mxnet-model-export --model-name="imperative-alexnet" --model-path="/tmp/models" --service-file-path="/tmp/models/gluon_imperative_alexnet.py" --model-type="imperative"
+```
+This creates a model-archive file `imperative-alexnet.model`.
+
+5. You could run the server with this model file to serve the pre-trained alexnet.
+```bash
+mxnet-model-server --models alexnet=imperative-alexnet.model
+```
+6. Test your service
+```bash
+curl -O https://s3.amazonaws.com/model-server/inputs/kitten.jpg
+curl -X POST http://127.0.0.1:8080/alexnet/predict -F "data=@kitten.jpg"
+```
+
+The output should be close to the following:
+
+```json
+{"prediction":[{"class":"lynx,","probability":0.9411474466323853},{"class":"leopard,","probability":0.016749195754528046},{"class":"tabby,","probability":0.012754007242619991},{"class":"Egyptian","probability":0.011728651821613312},{"class":"tiger","probability":0.008974711410701275}]}
+```
+
+## Load and serve a hybrid Gluon model
+
+To serve hybrid Gluon models with MMS, let's consider `gluon_imperative_alexnet.py` in `mxnet-model-server/examples/gluon_alexnet` folder.
+We first convert the model to a `Gluon` hybrid block. 
+For additional background on using `HybridBlocks` and the need to `hybridize` refer to this Gluon [hybridize](http://gluon.mxnet.io/chapter07_distributed-learning/hybridize.html#) tutorial.
+
+The above network, after this conversion, would look as follows: 
+```python
+class GluonHybridAlexNet(HybridBlock):
+    """
+    Hybrid Block gluon model
+    """
+    def __init__(self, classes=1000, **kwargs):
+        super(GluonHybridAlexNet, self).__init__(**kwargs)
+        with self.name_scope():
+            self.features = nn.HybridSequential(prefix='')
+            with self.features.name_scope():
+                self.features.add(nn.Conv2D(64, kernel_size=11, strides=4,
+                                            padding=2, activation='relu'))
+                self.features.add(nn.MaxPool2D(pool_size=3, strides=2))
+                self.features.add(nn.Conv2D(192, kernel_size=5, padding=2,
+                                            activation='relu'))
+                self.features.add(nn.MaxPool2D(pool_size=3, strides=2))
+                self.features.add(nn.Conv2D(384, kernel_size=3, padding=1,
+                                            activation='relu'))
+                self.features.add(nn.Conv2D(256, kernel_size=3, padding=1,
+                                            activation='relu'))
+                self.features.add(nn.Conv2D(256, kernel_size=3, padding=1,
+                                            activation='relu'))
+                self.features.add(nn.MaxPool2D(pool_size=3, strides=2))
+                self.features.add(nn.Flatten())
+                self.features.add(nn.Dense(4096, activation='relu'))
+                self.features.add(nn.Dropout(0.5))
```

Comment:
Why do we have dropout layers in the inference?
Will they be disabled when running inference? otherwise we should not include them and explain why we're doing it in comments.